flink-formats/flink-csv/pom.xml

```diff
@@ -77,6 +77,14 @@ under the License.
 
 		<!-- test dependencies -->
 
+		<dependency>
```

Comment:
Nit: Why was needed?

Reply:
Good point, it is a leftover from an earlier testing approach, not needed anymore.